src/ddmd/parse.d

```diff
@@ -4664,7 +4664,7 @@ final class Parser(AST) : Lexer
         case TOKlcurly:
             if (f.frequire || f.fensure)
                 error("missing body { ... } after in or out");
-            f.fbody = parseStatement(PSsemi);
+            f.fbody = parseStatement(0);
```

Comment:
I was going to make a new enum value `PSdefault` or so, but I saw `parseStatement` called with `0` in other places as well, so I left it like this.